### PR TITLE
DistanceVolume_CL

### PR DESCRIPTION
**Pull request by pingouinn**

### CODE REVIEW

File: client/html/index.html [hunk 1]
```diff
@@ -13,12 +13,18 @@
...
                     audioPlayer.pause();
                   }
-
                   audioPlayer = new Howl({src: ["./sounds/" + event.data.transactionFile + ".ogg"]});
                   audioPlayer.volume(event.data.transactionVolume);
                   audioPlayer.play();
-
                 }
+				
+				        if (event.data.transactionType == "volume") {
```
Comment (plunkettscott) [inline client/html/index.html]: Let's fix the alignment of these conditionals

File: client/main.lua [hunk 2]
```diff
@@ -77,16 +85,104 @@ end)
 RegisterNetEvent('InteractSound_CL:PlayWithinDistance')
 AddEventHandler('InteractSound_CL:PlayWithinDistance', function(otherPlayerCoords, maxDistance, soundFile, soundVolume)
 	if hasPlayerLoaded then
-		local myCoords = GetEntityCoords(PlayerPedId())
-		local distance = #(myCoords - otherPlayerCoords)
-
+        if tonumber(otherPlayerCoords) then
```
Comment (plunkettscott) [inline client/main.lua]: More alignment issues

**Review by plunkettscott — CHANGES_REQUESTED** [review]
Few minor house cleaning things, otherwise looks good for an experimental build.

File: client/main.lua (hunk 2 above)
Comment (plunkettscott) [reply client/main.lua]: Make sure the conditionals align with `distance` below.

**Comment by pingouinn** [comment]
I will do that, sorry for the cleaning problems i have made the commit with my phone haha !

**Pull request closed without merging**